app/workers/send_course_full_notifications_worker.rb

```diff
@@ -0,0 +1,34 @@
+class SendCourseFullNotificationsWorker
+  include Sidekiq::Worker
+
+  def perform
+    GetApplicationChoicesWithNewlyUnavailableCourses.call.each do |application_choice|
+      reason = ReasonCourseNotAvailable.new(application_choice).call
+      if FeatureFlag.active?(:unavailable_course_notifications)
+        ChaserSent.create!(
```

Comment:
Should this be outside of the conditional? Or we'll end up with constantly repeating slack messages, since they're never marked as sent?